### PR TITLE
Wouter martijn sdl facade

### PR DESCRIPTION
**Pull request by WoutervanGeel**
Added interface for SDL2 facade + color.hpp and font.hpp

### CODE REVIEW

File: src/SdlFacade.hpp [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+// Created by martijn on 11/1/16.
+//
+
+#ifndef MINOR_VIDYA_SDLFACADE_HPP
+#define MINOR_VIDYA_SDLFACADE_HPP
+
+#include <string>
+#include <map>
+#include <vector>
+#include "SDL2/SDL.h"
```
Comment (ThomasvLingen) [inline src/SdlFacade.hpp]: This comes from your system path, and should therefore be surrounded by <>s
Example: `#include <SDL2/SDL.h>`

File: src/SdlFacade.hpp [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+typedef std::vector<Key> PressedKeys;
+
+class SdlFacade {
+
+    private:
+        PressedKeys _keys_down;
+
+        map<FontType, TTF_Font *> _fonts;
+        map<SDL_Keycode *, Key> _possible_keys;
```
Comment (ThomasvLingen) [inline src/SdlFacade.hpp]: Why is SDL_Keycode a pointer?
You get SDL_Keycode objects, not pointers from the events

File: src/SdlFacade.hpp [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+using std::map;
+using std::vector;
+
+
+enum FontType {verdana, roman};
+enum Key {W, A, S, D};
+
+typedef std::vector<Key> PressedKeys;
+
+class SdlFacade {
```
Comment (ThomasvLingen) [inline src/SdlFacade.hpp]: Isn't SDL n acronym? Should it be `SDLFacade`?

File: src/SdlFacade.hpp (hunk 1 above)
Comment (WoutervanGeel) [inline src/SdlFacade.hpp]: done

File: src/SdlFacade.hpp (hunk 3 above)
Comment (WoutervanGeel) [inline src/SdlFacade.hpp]: done

File: src/SdlFacade.hpp (hunk 2 above)
Comment (WoutervanGeel) [inline src/SdlFacade.hpp]: done

**Pull request merged**